.github/actions/deploy-dsp/action.yml

```diff
@@ -0,0 +1,52 @@
+# Pre-condition: DSPO git repo checked out in appropriate branch
+# Kind cluster should be set up before hand
```

Comment:
Same as the previous comment, it looks like the steps in this yaml have been duplicated, are already listed out in the `ci-kind-tests.yml` workflow, and this yaml isn't being called. We should take it out if not being used.